substratevm/REFLECTION.md

```diff
@@ -13,8 +13,8 @@ where `reflectconfig` is a JSON file in the following format (use `--expert-opti
 	[
 	  {
 	    "name" : "java.lang.Class",
-      "allDeclaredConstructors" : true,
-      "allPublicConstructors" : true
+	    "allDeclaredConstructors" : true,
+	    "allPublicConstructors" : true
```

Comment:
missing comma?

Reply:
@panga This PR fix more mistakes on the same file: https://github.com/oracle/graal/pull/512